DokanNet/DokanHandle.cs

```diff
@@ -0,0 +1,22 @@
+﻿using DokanNet.Native;
+using Microsoft.Win32.SafeHandles;
+
+namespace DokanNet
+{
+    internal class DokanHandle : SafeHandleZeroOrMinusOneIsInvalid
```

Comment:
If you have time, it would be great to add some doc here (even a small one) since all the library is documented

Reply:
Yes, I'll fix some documentation tags!